test/test.robots.js

```diff
@@ -190,4 +190,37 @@ lab.experiment('hapi-robots', () => {
     Code.expect(response.payload).to.equal(str);
     Code.expect(response.headers['content-type']).to.include('text/plain');
   });
+
+  lab.test('supports sitemap as string', async() => {
+    await server.register({
+      plugin: robotModule,
+      options: {
+        verbose: true,
+        sitemap: 'http://somewhere.com/sitemap.xml'
```

Comment:
In the spec, do I have to have the full host? Or can just be a path? If full host, I'd like the library to figure that out for me